elb/src/main/java/org/jclouds/elb/features/AvailabilityZoneApi.java

```diff
@@ -27,7 +27,7 @@
 
 import org.jclouds.aws.filters.FormSigner;
 import org.jclouds.elb.binders.BindAvailabilityZonesToIndexedFormParams;
-import org.jclouds.elb.xml.AvailabilityZonesResultHandler;
+import org.jclouds.elb.xml.MemberResultHandler;
```

Comment:
I renamed this handler because its not specific to availability zones, so it can be used in other places e.g. the subnet responses